endpoints/users_id_token.py

```diff
@@ -672,7 +672,7 @@ def convert_jwks_uri(jwks_uri):
   return jwks_uri.replace(_TEXT_CERT_PREFIX, _JSON_CERT_PREFIX)
 
 
-def get_verified_jwt(issuers, audiences, cert_uri, cache=memcache):
+def get_verified_jwt(providers, audiences, cache=memcache):
```

Comment:
Given that this appears to be a public method, I think we should document the arguments with description + expected type.